docs/branches.md

```diff
@@ -138,6 +138,17 @@ Branches that already existed before the `jj git fetch` are not affected. This
 is similar to Mercurial, which fetches all its bookmarks (equivalent to Git
 branches) by default.
 
+## Branch movement
+
+Currently Jujutsu only moves branches by itself when these conditions are met:
+
+ * When a commit has been rewritten (so when you `jj split` a change) the 
+   working-copy will move along with it.
+ * When a commit has been abandoned, all associated branches will be moved 
+   with the working-copy to its parent(s).
+
+You could describe the movement as following along the change-id of the 
+working-copy commit, even if it isn't entirely accurate.
```

Comment:
As above, "of the working-copy commit" is confusing to me. Branches follow the change id even if there's no working copy pointing to it.

Reply:
I renamed "the working-copy commit" to the "current branch commit" to hopefully capture it a bit better.